zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/CamundaExporter.java

```diff
@@ -243,6 +243,10 @@ private void scheduleDelayedFlush() {
   }
 
   private void flushAndReschedule() {
+    if (configuration.getIndex().shouldWaitForImporters() && !importersCompleted) {
+      scheduleDelayedFlush();
+      return;
+    }
```

Comment:
❌ 🤔 Not sure whether this is the right fix. As this will no longer schedule more flushes. 

Can we maybe not schedule the first flush, and only schedule it if importing is done?